airbyte-integrations/connectors/source-lever-hiring/integration_tests/sample_config.json

```diff
@@ -4,4 +4,8 @@
   "refresh_token": "refresh_token",
   "environment": "Sandbox",
   "start_date": "2021-07-12T00:00:00Z"
+},
+{
+  "api_key": "c1adBeHUJKqyPM0X01tshKl4Pwh1LyPdmlorXjfmoDE0lVxZl",
```

Comment:
it's not safe to include a real API key here, can you change to another string. Thanks.

Reply:
It's not a real API key here, I took one and replaced characters randomly. I feel like it's better to have something that looks like one, to ensure a better comprehension, but as you wish. Let me know if you want me to change it, given this context